src/app/models/api-models/application/application.ts

```diff
@@ -17,7 +17,7 @@ interface ApplicationApiAttributes {
   invoice: Invoice;
   job: Job;
   jobEnded: boolean;
-  language: Language;
+  language: Language; // TODO missing relation in docs?
```

Comment:
What do you mean missing relation? :) I see it here: https://api.justarrived.se/api_docs/1.0/job_users.html